### PR TITLE
Fixed mso_schema read issue during the state file refresh

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Fixed mso_schema read issue during the state file refresh when the object is not present in the MSO/NDO but present in the state file.

### CODE REVIEW

**Review by shrsr — CHANGES_REQUESTED** [review]

File: mso/resource_mso_schema.go [hunk 1]
```diff
@@ -364,7 +364,12 @@ func resourceMSOSchemaRead(d *schema.ResourceData, m interface{}) error {
 	dn := d.Id()
 	con, err := msoClient.GetViaURL(fmt.Sprintf("api/v1/schemas/" + dn))
 	if err != nil {
-		return err
+		if con.S("code").String() == "404" {
```
[the remark this answers is not in the document]
Comment (sajagana) [reply mso/resource_mso_schema.go:367]: 1. This issue may be common for other resources.
2. MSO_Schema failure log: `{"code":404,"message":"Item not found"}` when we delete the object manually. Also attached the sample response value of the  MSO/NDO API.

**Review by shrsr — APPROVED** [review]
LGTM

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by akinross — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**